src/common/IPLocation/IPLocation.cpp

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (C) 2016+ AzerothCore <www.azerothcore.org>, released under GNU AGPL v3 license: https://github.com/azerothcore/azerothcore-wotlk/blob/master/LICENSE-AGPL3
+ * Copyright (C) 2021+ WarheadCore <https://github.com/WarheadCore>
```

Comment:
Should also include TC?

Reply:
i use https://github.com/WarheadCore/Warhead, fork TC